Add files via upload

trevorcummingsCCS opened the pull request:
Just putting files here to ensure they are available to all and obviously backed up

trevorcummingsCCS committed:
Add files via upload

trevorcummingsCCS committed:
Update populate_spree_tables.sql

Added spree_option_value_variants

trevorcummingsCCS committed:
Update populate_spree_tables.sql

added spree assets

tmckernan commented on shopping-service/create_mirror_spree_tables.sql:
```diff
@@ -0,0 +1,160 @@
+    updated_at timestamp without time zone NOT NULL,
+    "position" integer DEFAULT 0,
+    filterable boolean,
+    CONSTRAINT spree_taxonomies_pkey PRIMARY KEY (id)
```
this needs to be `CONSTRAINT load_spree_taxonomies_pkey PRIMARY KEY (id)`

tmckernan commented on shopping-service/create_mirror_spree_tables.sql:
```diff
@@ -0,0 +1,160 @@
+    depth integer,
+    hide_from_nav boolean DEFAULT false,
+    cnet_category_id character varying COLLATE pg_catalog."default",
+    CONSTRAINT spree_taxons_pkey PRIMARY KEY (id)
```
`CONSTRAINT load_spree_taxons_pkey PRIMARY KEY (id`

tmckernan commented on shopping-service/create_mirror_spree_tables.sql:
```diff
@@ -0,0 +1,160 @@
+    taxon_id integer,
+    id serial,
+    "position" integer,
+    CONSTRAINT spree_products_taxons_pkey PRIMARY KEY (id)
```
` CONSTRAINT load_spree_products_taxons_pkey PRIMARY KEY (id)`

tmckernan commented on shopping-service/create_mirror_spree_tables.sql:
```diff
@@ -0,0 +1,160 @@
+    created_at timestamp without time zone NOT NULL,
+    updated_at timestamp without time zone NOT NULL,
+    filterable boolean,
+    CONSTRAINT spree_properties_pkey PRIMARY KEY (id)
```
`CONSTRAINT load_spree_properties_pkey PRIMARY KEY (id)`

tmckernan commented on shopping-service/create_mirror_spree_tables.sql:
```diff
@@ -0,0 +1,160 @@
+    "position" integer DEFAULT 0,
+    "group" character varying COLLATE pg_catalog."default",
+    show_property boolean DEFAULT true,
+    CONSTRAINT spree_product_properties_pkey PRIMARY KEY (id)
```
`CONSTRAINT load_spree_product_properties_pkey PRIMARY KEY (id)`

tmckernan commented on shopping-service/create_mirror_spree_tables.sql:
```diff
@@ -0,0 +1,160 @@
+    updated_at timestamp without time zone NOT NULL,
+    "group" character varying COLLATE pg_catalog."default",
+    filterable boolean,
+    CONSTRAINT spree_option_types_pkey PRIMARY KEY (id)
```
`CONSTRAINT load_spree_option_types_pkey PRIMARY KEY (id)`

tmckernan commented on shopping-service/create_mirror_spree_tables.sql:
```diff
@@ -0,0 +1,160 @@
+	option_type_id integer NOT NULL,
+    created_at timestamp without time zone NOT NULL,
+    updated_at timestamp without time zone NOT NULL,
+    CONSTRAINT spree_option_values_pkey PRIMARY KEY (id)
```
`CONSTRAINT load_spree_option_values_pkey PRIMARY KEY (id)`

tmckernan commented on shopping-service/create_mirror_spree_tables.sql:
```diff
@@ -0,0 +1,160 @@
+    updated_at timestamp without time zone NOT NULL,
+    discontinue_on timestamp without time zone,
+    created_at timestamp without time zone NOT NULL,
+    CONSTRAINT spree_variants_pkey PRIMARY KEY (id)
```
`CONSTRAINT load_spree_variants_pkey PRIMARY KEY (id)`

trevorcummingsCCS committed:
Update populate_spree_tables.sql

removed part of where clause from spree_assets which was restricting it to one row.

trevorcummingsCCS committed:
Update populate_spree_tables.sql

Amended media type ids for spree_assets

trevorcummingsCCS committed:
Update populate_spree_tables.sql

removed reference to digita_meta_data_stage in the three spree_assets queries.

trevorcummingsCCS committed:
Update populate_spree_tables.sql

-- set the position to the same as the id for the table. This query could be eliminated by using the sequence which populates the table.
					
update load_spree_option_types set position = id;

trevorcummingsCCS committed:
Update populate_spree_tables.sql

Amneded code to use row_number () over (partition by column order by column) see code for example

trevorcummingsCCS committed:
Update populate_spree_tables.sql

Amended load_product_option_types for the position data attribute

trevorcummingsCCS committed:
Update populate_spree_tables.sql

Added indexes

trevorcummingsCCS committed:
Update create_mirror_spree_tables.sql

Added spree_assets

trevorcummingsCCS committed:
Update create_mirror_spree_tables.sql

fixed syntax errors

trevorcummingsCCS committed:
Update create_mirror_spree_tables.sql

amended fk names

trevorcummingsCCS committed:
Update create_tables.sql

added missing tables

trevorcummingsCCS committed:
Update copy_text_to_tables.sql

dded additional data

trevorcummingsCCS committed:
Update populate_spree_tables.sql

added conditional IF statements around DDL

trevorcummingsCCS committed:
Update populate_spree_tables.sql

had a not in when it should have been an in

Commit:
removed insertinto spree_assets for types 4,5,10,14

Commit:
made option type changes, so checking in

Commit:
option value variants

Commit:
scale_cnet_identities

Commit:
friendly_id_slugs

Commit:
removed references to load_spree

Commit:
bug fixes after first run into sbx3

Commit:
Put script to populate scale_product_xmls into populate_spree_tables.sql

Commit:
Update spree_assets so only one image for CCS Product Image is picked

Commit:
Update spree_assets positions

Commit:
Updated for index creation in the script

Commit:
Added weight update to spree_variants

Commit:
Added simple logging

Commit:
Added list content guids not to be added

tommyb82 commented:
Redundant - closing PR
